### ISSUE TITLE
:file being ignored, actual image in ob-ipython-resources-dir not displayed

### ISSUE THREAD

**Issue by eludom** [gregsexton/ob-ipython#121]
It seems that the file specified in :file parameter (see except from README below) properly 
puts the :file parameter value in the file, but the image is being generated in  ob-ipython-resources-dir
is not being displayed.    Generate using :file specified name if provided?

     #+BEGIN_SRC ipython :session :file /tmp/image.png :exports both :results raw drawer
       _ = plt.hist(np.random.randn(20000), bins=200)
     #+END_SRC

     #+RESULTS:
     :RESULTS:
     [[file:/tmp/image.png]]
     :END:

`

**Comment by eludom** [comment]
:file works with async

**Comment by gregsexton** [comment]
This was painful! Org overrides the output of ob-ipython if you provide a :file param. It was just coincidence that it worked before unfortunately. I've solved the problem by ignoring :file and instead honouring :ipyfile. See the updated readme for examples (and if you care about the details I've explained them in the recent commit).

Hopefully this fixes the issue for you. Not the most elegant solution but I didn't have much choice.
